index.html

```diff
@@ -5,11 +5,147 @@
+            <h1>colorado life</h1>
+    </header>
+    </div>
+    <div class="morning">
+        <h2>Morning</h2>
+        <div class="morningphoto">
+            <figure><img src="images/dog.jpg" alt="dog next to the bed">
+                <figcaption class="time"> 9:00 AM</figcaption>
+                <figcaption>When I woke up in the morning, the dog lie down on the floor next to the bed.
+                    Who's name is Sparky.
+                </figcaption>
+            </figure>
+        </div>
+        <div class="morningphoto">
+            <figure><img src="images/morning_drive.jpg" alt="morning drive">
+                <figcaption class="time"> 10:00 AM</figcaption>
+                <figcaption>My friend drove us to the Colorado Springs. Our destination was "pikes park" which is the
+                    national park in Colorado. </figcaption>
+            </figure>
+        </div>
+        <div class="morningphoto">
+            <figure><img src="images/morning_road.jpg" alt="morning road">
+                <figcaption class="time"> 11:00 AM</figcaption>
+                <figcaption>
+                    I was impressed by the landscape I saw during the driving. The moutains were all around the
+                    Colorado.</figcaption>
+            </figure>
+        </div>
+    </div>
+
+
+    <figure><img class="full" src="images/god.jpg" alt="garden of the gods">
+        <figcaption class="time2"> 12:00 PM</figcaption>
+        <figcaption>I took a bus tour and passed the Garden of the Gods park, which is famous national park in Colorado.
+            There are a lot of beautiful red rocks. We didn’t have chance to get off the bus and take a close look but I
+            saw the famous rock called kissing camels.
+        </figcaption>
+        <figcaption>Bus driver who was very knowledgeable explained us why they called by that name. The top part of the
+            rock
+            looks like two camels are kissing so they called kissing camel. The kissing camel is located at the very top
+            of North Gateway Rock so we could easily see it.
+        </figcaption>
+    </figure>
+    <figure><img class="full" src="images/snow.jpg" alt="snow mountain">
+        <figcaption class="time2"> 1:00 PM</figcaption>
+        <figcaption>I was surprised that there were still snow at the top of the mountains. Because of the weather
+            problem, we could not go higher but it was enough to see this beautiful view.
+        </figcaption>
+        <figcaption>Me and my friend took a lot of picture and have a lot of snow on the surface, my friend could do
+            snow angel
+            on the snow. I really love this experience because this was the first time I have been to the mountain in
+            U.S.
+        </figcaption>
+    </figure>
+
+    <div class="afternoon">
+        <h2>Afternoon</h2>
+        <div class="afterphoto">
+            <figure><img src="images/boulder.jpg" alt="street view of boulder">
```

Comment:
Note that your images have not been resized. This image is 4032px wide, but is only displayed in your layout at ~700px wide.